config/crds/starlingx_v1_host.yaml

```diff
@@ -670,6 +670,11 @@ spec:
                   description: Location defines the physical location of the host
                     in the data centre.
                   type: string
+                maxCPUFrequency:
+                  description: MaxCPUFrequency defines the maxium limit of the CPU
+                    frequency set on the host. + optional
+                  pattern: ^[1-9][0-9]*$
```

Comment:
As an FYI, this means you can never set this to zero.

Reply:
As per Iago, the max CPU frequency should always be above 0